kubectl-volsync/cmd/migration_create.go

```diff
@@ -17,14 +17,69 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 package cmd
 
 import (
+	"context"
 	"fmt"
+	"time"
 
 	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	kerrs "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
 	"k8s.io/kubectl/pkg/util/i18n"
 	"k8s.io/kubectl/pkg/util/templates"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	volsyncv1alpha1 "github.com/backube/volsync/api/v1alpha1"
 )
 
+type migrationCreate struct {
+	// migration relationship object to be persisted to a config file
+	mr *migrationRelationship
+	// contains replication destination metadata
+	replicationDestinationMetadata
+	// contains replication destination specs
+	replicationDestinationRsyncSpec
```

Comment:
If these structs aren't used anywhere else, please just inline the fields.

Reply:
sure